File: javascript/pangram/pangram.js
function Pangram(input) {
  var chars = input.toLowerCase().replace(/\W|\d|_/g, "").split("");
  this.isPangram = function() {
    return chars.filter((e, i, self) => self.indexOf(e) === i).length === 26;
  };
};

module.exports = Pangram;

File: csharp/pangram/Pangram.cs
using System.Linq;
using System.Text.RegularExpressions;

class Pangram
{
	public static bool IsPangram(string phrase)
	{
		phrase = Regex.Replace(phrase.ToLower(), @"\W|\d|_|[^\u0000-\u007F]+", "");
		return phrase.Where((x, i) => phrase.IndexOf(x) == i).Count() == 26;
	}
}
